src/main/java/Event.java

```diff
@@ -0,0 +1,46 @@
+public class Event extends Task {
+    protected String from;
+    protected String to;
+    public Event(String taskDescription, String from, String to) {
+        super(taskDescription);
+        this.taskType = TaskType.EVENT;
+        this.from = from;
+        this.to = to;
+    }
+
+    public String getFrom() {
+        return this.from;
+    }
+
+    public String getTo() {
+        return this.to;
+    }
+
+    @Override
+    public void printAdd(int todosCount) {
+        System.out.println("Got it! I've added this task: ");
+        System.out.println("  [E][ ] "
+                + this.getTaskDescription()
+                + "(from: "
+                + this.from
+                + "to: "
+                + this.to
+                + ")");
+        System.out.println("Now you have " + todosCount + " tasks in your list.");
+    }
+
+    @Override
+    public void printTask(int todoCount) {
+        System.out.println((todoCount + 1)
+                + ".[E]"
+                + "["
+                + (this.getTaskStatus() ? "X" : " ")
+                + "] "
+                + this.getTaskDescription()
+                + "(from: "
+                + this.from
+                + "to: "
+                + this.to
+                + ")");
```

Comment:
Creating a new line for every '+' string operands causes the overall code to be harder to read so I believe it's better to just make combine it within 2-3 lines.